docs/azure-vpn-db1000n.md

```diff
@@ -0,0 +1,114 @@
+Їх буде достатньо щоб запустити 2 виділених сервери строком на 1 місяць.
+
+## Регістрація на Microsoft Azure
+
+Для реєстрації буде потрібно діючий номер телефону та банківська карта (може підійти наприклад інтернет-карта привату).
+
+Ідемо на сайт https://azure.microsoft.com/ і жмемо **start free**
+
+<img src="img/Screenshot_1.png">
+
+Далі вам потрібно буде увійти в обліковий запис microsoft або створити його якщо в вас його немає. 
+
+<img src="img/Screenshot_2.png">
+
+В процесі реєстрації вписуємо номер телефону і на нього прийде код підтвердження,
+
+також вписуємо дані карти - з неї спишеться і потім повернеться 1$
+
+З іншим при реєстрації не повинно виникнути проблем. якщо ви не можете зарєєструватися на сайті - то що ви тут робите? )))
+
+## Створення virtual mashines
+
+Далі ви повинні попасти на панель керування https://portal.azure.com/#home
+
+Натискаємо створити віртуальну машину
+
+<img src="img/Screenshot_3.png">
+
+Створюємо Resourse group якщо іще не створена, даємо назву віртуальній машині, вибираємо регіон в якому буде знаходитися наш сервер (тут можуть бути різні варіанти, вибирайте самі)
+Далі обираємо систему яка буде встановлена. нам буде достатньо windows 10 pro за 70$ в місяць. 
+
+<img src="img/Screenshot_4.png">
+
+Далі заповнюємо логін і пароль адміністратора (там є вимоги - дивиомсь підказки) та обираємо порт для підключення до виділенного серверу. порт обирайте **RDP(3389)**
+ставимо галочку ліцензії
+натискаємо кнопку **Review + create**  і чекаємо коли буде створена наша віртуальна машина.
+
+<img src="img/Screenshot_5.png">
+
+Коли буде створена переходимо до параметрів натиснувши кнопку **Go to resource**
+
+<img src="img/Screenshot_6.png">
+
+далі натискаємо **connect - RDP**
+
+<img src="img/Screenshot_7.png">
+
+і скачуємо файл натиснувши на кнопку **Download RDP File**
+
+<img src="img/Screenshot_8.png">
+
+запускаємо скачаний файл 
+ 
+ <img src="img/Screenshot_9.png">
+ 
+вводимо логін і пароль адміністратора які ми писали при створвенні віртуальної машини
+
+<img src="img/Screenshot_10.png">
+
+підтверджуємо сертифікати
+
+<img src="img/Screenshot_11.png">
+
+і далі відкриється вікно першого входу на встановлений віндовс. нажимаємо **Accept**
+
+<img src="img/Screenshot_12.png">
+
+Вітаю вас - віртуальна машина створена!
+
+
+
+Далі потрібно встановити впн та программу db1000n
+
+## Встановлення VPN
+
+Тут э величезна варіативність. VPN ви можете встановлювати такий який подобається вам. Я буду встановлювати на прикладі **dewVPN**
+
+На виділеному сервері заходимо в браузер і відкриваємо https://www.dewvpn.com/ скачуємо та встановлюємо. 
+
+нажимаємо choose location - вибираємо russia і нажимаємо ON
+якщо російські сервери не запускається - вибираємо іншу країну.
+
+<img src="img/Screenshot_13.png">
+
+## Встановлення програми db1000n 
+
+Далі заходимо на сторінку https://github.com/Arriven/db1000n гортаємо вниз і скачуємо версію для windows 
```

Comment:
There is a script which download and run app https://github.com/Arriven/db1000n/blob/main/scripts/StartDB1000N.bat